src/renderer.rs

```diff
@@ -320,4 +325,71 @@ impl Renderer {
             line(color::BLACK.into(), 1., [start[0], start[1], end[0], end[1]], c.transform, g);
         }
     }
+
+    /// Save the drawings to SVG
+    fn save_svg(&mut self, path: &path::Path) {
+        let drawing_state = self.app.drawing();
+        let mut document = svg::Document::new()
+            .set("viewBox", (0, 0, drawing_state.width, drawing_state.height));
+
+        // set background color - https://stackoverflow.com/a/11293812/9276882
+        let bg_color = drawing_state.background;
+        let background = SvgRect::new()
+            .set("width", "100%")
+            .set("height", "100%")
+            .set("fill", bg_color.rgba_string());
+
+        document = document.add(background);
+
+        let center = drawing_state.center.to_screen_coords(
+                        Point {
+                            x: drawing_state.width as f64 * 0.5,
+                            y: drawing_state.height as f64 * 0.5,
+                        });
```

Comment:
nit: these lines can probably be formatted closer to the other instance of this in this file:

https://github.com/sunjay/turtle/blob/14f882bc7fed96a9498e715c4d87c8e797233a31/src/renderer.rs#L128-L131